eng/helix/content/runtests.cmd

```diff
@@ -4,35 +4,37 @@ setlocal enabledelayedexpansion
 
 REM Use '$' as a variable name prefix to avoid MSBuild variable collisions with these variables
 set $target=%1
-set $sdkVersion=%2
-set $runtimeVersion=%3
-set $queue=%4
-set $arch=%5
-set $quarantined=%6
-set $ef=%7
-set $aspnetruntime=%8
-set $aspnetref=%9
-REM Batch only supports up to 9 arguments using the %# syntax, need to shift to get more
-shift
+set $runtimeVersion=%2
+set $queue=%3
+set $arch=%4
+set $quarantined=%5
+set $ef=%6
+set $aspnetruntime=%7
+set $aspnetref=%8
 set $helixTimeout=%9
+REM Batch only supports up to 9 arguments using the %# syntax, need to shift to get more
 
-set DOTNET_HOME=%HELIX_CORRELATION_PAYLOAD%\sdk
-set DOTNET_ROOT=%DOTNET_HOME%\%$arch%
+set DOTNET_ROOT=%HELIX_CORRELATION_PAYLOAD%\dotnet
+set DOTNET_HOME=%DOTNET_ROOT%
 set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 set DOTNET_MULTILEVEL_LOOKUP=0
-set DOTNET_CLI_HOME=%HELIX_CORRELATION_PAYLOAD%\home
+set DOTNET_CLI_HOME=%DOTNET_ROOT%\home
 
 set PATH=%DOTNET_ROOT%;!PATH!;%HELIX_CORRELATION_PAYLOAD%\node\bin
 echo Set path to: %PATH%
-echo "Invoking InstallDotNet.ps1 %$arch% %$sdkVersion% %$runtimeVersion% %DOTNET_ROOT%"
-powershell.exe -NoProfile -ExecutionPolicy unrestricted -file InstallDotNet.ps1 %$arch% %$sdkVersion% %$runtimeVersion% %DOTNET_ROOT%
+
+echo "Invoking InstallDotNet.ps1 %$arch% %$runtimeVersion% %DOTNET_ROOT%"
+powershell.exe -NoProfile -ExecutionPolicy unrestricted -file InstallDotNet.ps1 %$arch% %$runtimeVersion% %DOTNET_ROOT%
```

Comment:
Writing to a path under %HELIX_CORRELATION_PAYLOAD% will continue to not work in docker-ized Helix queues.  If you plan to change it in the future this may be a fine workaround but it probably merits a comment letting the next person to notice the read-only-ness in Docker know what's up.

Reply:
Are there going to be windows based docker queues in the future?  Today we only seem to running into issues on the .sh scripts

Reply:
But sure i'll add a comment here